### ISSUE TITLE
[please add] FTB Tutorials

### ISSUE THREAD

**Issue by iamweirdie** [kreezxil/World-of-Dragons#315]
https://www.curseforge.com/minecraft/mc-mods/ftb-tutorial-mod

I'd like to be more blatant about forcing players to read at least a LITTLE at the beginning before hitting admins with FAQs such as "how do i tinkers construct"

This mod also allows creators to import pretty pictures, in case quests were too wordy for players to follow.

**Comment by IppSwitch** [comment]
Brilliant idea. I agree to this purely because over the last couple of days i have noticed an increased amount of hoppers in new players bases. People seem to not read the quest book or don't know it's there.

**Comment by hellwolf95** [comment]
Tell them there is a god damn book on it in game instead of crying about it. Not to mention guide book can do the same thing so i dont think kree will add this.

**Comment by kreezxil** [comment]
Exactly, don't just notice that players are ignoring stuff, point it out to
them. FTB TUTORIALS will be just another thing for them to ignore.

**Comment by kreezxil** [comment]
also not gonna add this, did you even look at how complicated latvian modder made this?

I would add it if there was a way for me to drag and drop assets after the resources have been loaded to include placing text on the screen. 

On top of that, getting to the Tutorials is no more easier than it is to getting to the quests and the quests are more rewarding plus easier to create.

you sill have to tap 'e' and you still have move your eyes and the mouse to the upper left corner.

this is not easier by a long shot.
